src/main/java/uk/gov/hmcts/cmc/claimstore/rules/ClaimAuthorisationRule.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.cmc.claimstore.rules;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.exceptions.ForbiddenActionException;
+import uk.gov.hmcts.cmc.claimstore.idam.models.UserDetails;
+import uk.gov.hmcts.cmc.claimstore.services.UserService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+@Component
+public class ClaimAuthorisationRule {
+
+    private static final String USER_NOT_LINKED_MESSAGE =
+        "User with id %s1 is not linked to claim with external id %s2";
+
+    private static final String USER_ID_MISMATCH =
+        "User with id %s1 does not match given user id %s2";
```

Comment:
I think we should not be exposing the information of correct user id for the case here. Probably more generic message similar to the one we have in `USER_NOT_LINKED_MESSAGE`.